v1.10/cdk/PRODUCT.yaml

```diff
@@ -0,0 +1,6 @@
+vendor: Canonical Ltd
+name: Canonical Distribution of Kubernetes
+version: v1.10.0
+website_url: https://www.ubuntu.com/kubernetes
+documentation_url: https://kubernetes.io/docs/getting-started-guides/ubuntu/
+product_logo_url: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.svg
```

Comment:
Please update the logo, as that belongs to Kubernetes, not the Canonical Distribution of Kubernetes.

We are currently using:
https://landscape.cncf.io/logos/canonical-distribution-of-kubernetes.svg
https://landscape.cncf.io/grouping=organization&organization=canonical&selected=canonical-distribution-of-kubernetes

We can continue to do so, if you want.

Reply:
Thank you for spotting this out. Just updated the logo as requested.

Reply:
Although the info displayed in https://landscape.cncf.io/grouping=organization&organization=canonical&selected=canonical-distribution-of-kubernetes are not right. Canonical Headcount is 500+ and the linkedin link is broken.

Reply:
@ktsakalozos The landscape pulls the Canonical info from: https://www.crunchbase.com/organization/canonical-ltd

I just updated the LinkedIn link. If you want to tell me the correct headcount, I can adjust that too, or you can do it yourself. We'll then fetch the corrected info tonight.

Reply:
Thank you, we will update the canonical ourselves.